NEWS.md

```diff
@@ -7,6 +7,10 @@ shiny 1.7.2.9000
 
 ### New features and improvements
 
+* `plotOutput()` and `imageOutput()` gain a `fill` argument. If `TRUE` (the default for `plotOutput()`), the output container is allowed to grow/shrink to fit a fill container (`htmltools::asFillContainer()`) with an opinionated height. Most importantly, this means `plotOutput()` will grow/shrink by default [inside of `bslib::card_body_fill()`](https://rstudio.github.io/bslib/articles/cards.html#responsive-sizing). (#3715) 
+
+* `uiOutput()` gains `fill` and `fillItem` arguments. Set `fill = TRUE` to treat the `uiOutput()`'s `container` as a fill container (`htmltools::asFillContainer()`), allowing it's contents to grow/shrink when the `container` has an opinionated height. (#3715)
```

Comment:
Is it just me or do the `fill` arguments in `plotOutput`/`imageOutput` vs. `uiOutput` have opposite meanings? In the former, it means "use `asFillItem()` on me" and in the latter, it means "use `asFillContainer()` on me"?

Reply:
It's not just you...it was intentional (for reasons I explain below)